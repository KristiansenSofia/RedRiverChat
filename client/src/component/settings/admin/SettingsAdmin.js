import React, { Component } from 'react'

// Import styles. settingsAdminStyles for all imported components with a style attribute and CSS-file for classNames and id.
//import {settingsAdminStyles} from '../../../styles/SettingsStyles'
import '../../../styles/Styles.css'

/**
 *  SettingsAdmin-component.
 *
 *  @author Jimmy
 */

class SettingsAdmin extends Component {
  render () {
    return (
      <div className='SettingsAdmin'>
        <p>Admin settings</p>
      </div>
    )
  }
}

export default SettingsAdmin
