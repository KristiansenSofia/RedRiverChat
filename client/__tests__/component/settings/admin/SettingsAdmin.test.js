import React from 'react';
import SettingsAdmin from '../../../../src/component/settings/admin/SettingsAdmin';
import renderer from 'react-test-renderer';

test('SettingsAdmin renders without crashing', () => {
  const tree = renderer.create(<SettingsAdmin />);
  expect(tree.toJSON()).toMatchSnapshot();
});
