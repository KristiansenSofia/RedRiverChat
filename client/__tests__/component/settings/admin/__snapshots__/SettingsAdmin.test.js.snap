// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsAdmin renders without crashing 1`] = `
<div
  className="SettingsAdmin"
>
  <p>
    Admin settings
  </p>
</div>
`;
